cluster/examples/kubernetes/rook-operator.yaml

```diff
@@ -37,12 +37,23 @@ rules:
   - deployments
   - daemonsets
   - replicasets
+  - statefulsets
```

Comment:
why is the `statefulset` resource added in two different groups?

Reply:
Because `statefulset` is/was in both resources, I think, depending on which K8s version you have see: https://kubernetes.io/docs/reference/workloads-18-19/.
K8s 1.8 had statefulset in `apps` and `extensions`.